### PR TITLE
Add config to override default response-header-size for HTTP client

### PR DESCRIPTION
**Pull request by elonlo**
To solve long prepare statement query failing with 500 error

reference to https://kevinyin.github.io/Priest/presto-long-prepareStatement-

### CODE REVIEW

File: http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java [hunk 1]
```diff
@@ -121,6 +122,7 @@ public void testExplicitPropertyMappings()
                 .put("http-server.admin.threads.min", "3")
                 .put("http-server.admin.threads.max", "4")
                 .put("http-server.max-request-header-size", "32kB")
+                .put("http-server.max-response-header-size", "32kB")
```
Comment (dain) [inline http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java]: Can you make this `57k`?  That is different from the value above and unlikely to ever be a default.

File: http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java [hunk 2]
```diff
@@ -160,6 +162,7 @@ public void testExplicitPropertyMappings()
                 .setThreadMaxIdleTime(new Duration(10, MINUTES))
                 .setNetworkMaxIdleTime(new Duration(20, MINUTES))
                 .setMaxRequestHeaderSize(new DataSize(32, KILOBYTE))
+                .setMaxResponseHeaderSize(new DataSize(57, KILOBYTE))
```
Comment (elonlo) [inline http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java]:
```suggestion
                .setMaxResponseHeaderSize(new DataSize(57, KILOBYTE))
```

File: http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java (hunk 1 above)
Comment (elonlo) [reply http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java]: okay, I have modified

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**